openedx/openedx-platform
# feat: adds Library Content button to Studio Unit page  [FC-0062]

Built on top of: https://github.com/open-craft/edx-platform/pull/699 and https://github.com/openedx/edx-platform/pull/35636

Depends on 
* https://github.com/openedx/frontend-app-authoring/pull/1401
* https://github.com/openedx/frontend-app-authoring/pull/1412

**First iteration**

https://github.com/user-attachments/assets/aa4e2ad9-5745-4fcb-94b1-a0ccab794665

**Second iteration**

https://github.com/user-attachments/assets/776403dc-a865-4568-a74a-809786c50efa


Test instructions:
1. Check out this branch
2. Run `tutor dev exec lms openedx-assets build --env=dev` to rebuild JS files
3. Check out https://github.com/openedx/frontend-app-authoring/pull/1401 in authoring MFE
4. Disable browser cache and view the unit page
